yarn.lock

```diff
@@ -5677,14 +5731,34 @@ ses@^0.18.4, ses@^0.18.5, ses@^0.18.8:
   dependencies:
     "@endo/env-options" "^0.1.4"
 
+ses@^1.3.0:
```

Comment:
Your screenshot of this working is evidence otherwise, but should we add a resolution for `ses`? 

```
diff --git a/package.json b/package.json
index 9e23e12..4bc7405 100644
--- a/package.json
+++ b/package.json
@@ -59,6 +59,7 @@
   },
   "resolutions": {
     "**/@agoric/xsnap": "0.14.3-dev-9f085d3.0",
-    "**/@agoric/time": "0.3.3-dev-97cb715.0"
+    "**/@agoric/time": "0.3.3-dev-97cb715.0",
+    "**/ses": "1.3.0"
   }
 }
 ```

Reply:
`yarn why ses` shows the old version is just comping up in transitive deps. The `@endo/init` runs at start so that's the only version that affects the runtime env.